TransbankPosSDK/POSAutoservicio.cs

```diff
@@ -0,0 +1,184 @@
+﻿using System;
+using System.Collections.Generic;
+using Transbank.Utils;
+using Transbank.Exceptions.CommonExceptions;
+using System.Text;
+using Transbank.Responses.CommonResponses;
+using Transbank.Responses.AutoservicioResponse;
+using Transbank.Exceptions.AutoservicioExceptions;
+using System.Threading.Tasks;
+
+namespace Transbank.POSAutoservicio
+{
+    public class POSAutoservicio : Serial
+    {
+        public POSAutoservicio()
+        {
+
+        }
+
+        public static POSAutoservicio Instance { get; } = new POSAutoservicio();
+
+        public async Task<bool> Poll()
+        {
+            DiscardBuffer();
+
+            if (CantWrite())
+            {
+                throw new TransbankException($"Unable to Poll port {Port.PortName} is closed");
+            }
```

Comment:
I think that all the methods should make this check. What do you think?

Reply:
It is always checked, it just depends on the method.

Reply:
I dont see the `if (canWrite())` on other methods

Reply:
In the serial class in the method WriteData